apps/account-api/src/main.ts

```diff
@@ -48,7 +48,20 @@ async function bootstrap() {
   const swaggerDoc = await generateSwaggerDoc(app, {
     title: 'Account Service',
     description: 'Account Service API',
-    version: '1.0',
+    version: '2.0',
+    extensions: new Map<string, any>([
+      [
+        'x-tagGroups',
+        [
+          { name: 'accounts', tags: ['v1/accounts', 'v2/accounts'] },
+          { name: 'delegation', tags: ['v1/delegation'] },
+          { name: 'delegations', tags: ['v2/delegations'] },
```

Comment:
Would we want to merge these just for the sake of clarity of joining or does it muddy the water too much?

```suggestion
          { name: 'delegations', tags: ['v1/delegation', 'v2/delegations'] },
```

Reply:
Hmmm... for grouping/docs purposes, it might be nice to have them grouped together, but it might cause some confusion. Though the actual OpenAPI spec would still be clear; it would only be in the doc grouping.

I think I like it... let me know if I should proceed with that.